manager/src/main/resources/application.properties

```diff
@@ -162,3 +162,12 @@ quarkus.log.console.json.fields.thread-id.enabled=false
 quarkus.log.console.json.fields.sequence.enabled=false
 # MGDOBR-453: Enabled logging for Workers
 quarkus.log.category."com.redhat.service.smartevents.manager.workers".level=DEBUG
+
+# MGDOBR-1009: kafka configuration for poll based error handling (errors are read from global kafka topic)
+kafka.bootstrap.servers=${event-bridge.kafka.bootstrap.servers}
```

Comment:
Won't these override the Error Handler queue used to connect Managed Connectors to RHOSE?

Reply:
Nope. The `manager` wasn't connecting to kafka up to this point. There's no conflict :+1: